### DISCUSSION TITLE
Cookie suffix "token"

### DISCUSSION THREAD

**Discussion by rishi-afk** [payloadcms/payload#379] [Feature Requests & Ideas]
Hello devs and maintainers, can we strip off the cookie suffix `token` [here](https://github.com/payloadcms/payload/blob/2faade7a0335c3f984b7fdf4c67b6aac75d1b09c/src/auth/operations/login.ts#L150) and just use the value passed in config as the cookie name? I was facing issues because of the `token` suffix while working with stripe react package.

**Comment by jmikrut** [comment]
Hey @rishi-afk — can you elaborate as to why the cookie suffix impacts usage of the Stripe React package?

We may be able to provide a mechanism to completely take over the cookie name, rather than just be able to set its prefix. But first I'd like to make sure that I understand why it conflicts with Stripe's React package.

Mind expanding on that a bit?
